packages/mobile/src/screens/profile-screen/ProfileHeader/SocialLink.tsx

```diff
@@ -58,23 +57,6 @@ export const SocialLink = (props: SocialLinkProps) => {
     setIsActive(false)
   }, [])
 
-  // undefined equates to "LOADING" from backend
```

Comment:
no longer an undefined / loading state as DNs will return either the social handle string or null

Reply:
still necessary because of the potential for fallback?

Reply:
no because it's never going to be undefined. DN would return null if it doesn't have it

Reply:
but if DN doesn't have it, and identity handles the request, can't it be undefined still?

Reply:
no. still null

Reply:
dn will always return something. null or string